app/javascript/components/recipes/new/list.vue

```diff
@@ -88,6 +89,7 @@ export default {
   data() {
     return {
       steps: [],
+      ingredients: this.elements,
```

Comment:
Clonaría el arreglo, creo que pueden aparecer warnings y comportamientos raros si usamos el mismo objeto en el data que en la prop:

```suggestion
      ingredients: [...this.elements],
```

Reply:
Entiendo perfecto, lo arregle de otra forma, porque entendi que estaba mal tenerlo en data y modificarlo ahi. Al final termine mandando un emit para que se modifique la data del padre

Reply:
Buena! Mucho mejor, quedo bacán 💯